src/resources/audit/AuditList.tsx

```diff
@@ -191,7 +195,7 @@ export default function AuditList({
           label='Subject'
           render={(record) => {
             return (
-              <SourceField
```

Comment:
@TahaKhanAbdalli - while you are working in this file, can I suggest you use the resourcesKeyRef index in the next SourceField like you do in the previous SourceField?  
That's more versatile than the hard-coded R_ITEMS check.